src/pages/SignUp/SignUp.scss

```
@@ -1,2 +1,106 @@
-* {
+.wholePage {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+
+  .Nav {
+    height: 80px;
+  }
+
+  .signUpContents {
+    display: flex;
+    flex-direction: column;
+
+    .signUpTitle {
+      width: 425px;
+      height: 23px;
+      margin: 40px 0 40px 0;
+      text-align: center;
+      font-weight: 500;
+      font-size: 17px;
+      letter-spacing: 0.6px;
+    }
+
+    .indicater {
+      width: 415px;
+      height: 20px;
+      margin: 7.8px 0 12px 0;
+      font-size: 13px;
+      opacity: 0.6;
+    }
+  }
+
+  .signUpInput {
+    display: flex;
+    flex-direction: column;
+    width: 415px;
+    height: 71.094px;
+    margin-bottom: 50px;
+
+    .info {
+      margin: 0 2px 2px 1.5px;
+      font-size: 12px;
+      font-weight: 700;
+      letter-spacing: 0.08em;
+
+      .dot {
+        margin-left: 4px;
+      }
+    }
+
+    .userInput {
+      width: 415px;
+      height: 43.09px;
+      padding: 1.13em;
+      margin-top: 8px;
+      border: 1px solid #d3d3d3;
+    }
+  }
+
+  .signUpBtn {
+    width: 250px;
+    height: 38px;
+    margin: 22px 0 30px 0;
+    border: 1px solid #000;
+    cursor: pointer;
+
+    .btnText {
+      font-size: 11px;
+      font-weight: 600;
+      letter-spacing: 0.5px;
+    }
+  }
+
+  .linkContain {
+    margin: 10px 0 18px 0;
+    font-size: 13px;
+    color: #0a0101ab;
+
+    a:link {
+      color: black;
+      text-decoration: underline;
+    }
+    a:visited {
+      color: black;
+      text-decoration: underline;
+    }
+    a:hover {
+      color: black;
+      text-decoration: underline;
+    }
```

```suggestion
a {
  &:hover, &:visited, &:link {
    color: black;
    text-decoration: underline;
  }
}
```
이런 식으로 작성하시면 좋을 것 같고 만약 이 컴포넌트 말고도 적용되어야 하는 내용이라면 reset, common 등으로 옮겨야 할 것 같습니다

---

> 이런 식으로 작성하시면 좋을 것 같고 만약 이 컴포넌트 말고도 적용되어야 하는 내용이라면 reset, common 등으로 옮겨야 할 것 같습니다

이 컴포넌트에만 적용되는 사항인 것 같아 위의 수정사항으로 수정 후 Sign Up.scss 에 유지하기로 결정했습니다.